### DISCUSSION TITLE
Can I bypass the google credentials

### DISCUSSION THREAD

**Discussion by dnjooiopa** [fsouza/fake-gcs-server#1118] [Q&A] [answered]
I want to run some tests in github action that upload and check some files in fake-gcs-server (running in another server)  but It required google credentials.

Is there any method to skip google credential checking?

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/34341154/229781723-0f68e744-7117-4191-bfbd-1aeff522849c.png">

**Comment by fsouza** [comment, marked as answer]
@dnjooiopa which language are you using? You can check the [examples](https://github.com/fsouza/fake-gcs-server/tree/HEAD/examples) folder for code samples in different languages that bypass Google creds.

**Reply by dnjooiopa** [reply]
@fsouza Thank you! I use golang. May be I need to check more in examples.

**Reply by Schachte** [reply]
tl;dr use `option.WithoutAuthentication()`. However! You must not pass in the credential file or JWT _in addition to_ the without auth flag or it'll fail.
